server/apps/publicapi_publish/packages.py

```diff
@@ -12,7 +12,10 @@
 
 
 class PublicPackagesResource(PackagesResource):
-    datasource = {'source': 'items'}
+    datasource = {
+        'search_backend': 'elastic',
+        'source': 'items'
+    }
     item_methods = ['DELETE', 'PATCH', 'GET', 'DELETE']
```

Comment:
This is the superdesk public API publish service, not the public API delivery service.